kalite/contentload/management/commands/generate_assessment_zips.py

```diff
@@ -56,14 +62,14 @@ def download_urls(zf, urls):
 
     urls = set(urls)
 
-    pool = ThreadPool(5)
+    pool = ThreadPool(10)
```

Comment:
Looks like a mutex (see below) fixes it beautifully. Let there be concurrency! :confetti_ball: